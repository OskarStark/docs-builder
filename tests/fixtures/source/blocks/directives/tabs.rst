
.. tabs:: UX Installation

    .. tab:: Webpack Encore

        Webpack Encore stuff!

        .. code-block:: yaml

            I am yaml:

    .. tab:: AssetMapper

        AssetMapper stuff!

        And another paragraph.
