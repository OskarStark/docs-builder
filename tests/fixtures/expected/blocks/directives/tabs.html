<div class="configuration-block">
    <div role="tablist" aria-label="UX Installation" class="configuration-tabs configuration-tabs-length-2">
        <button role="tab" type="button" data-language="webpack-encore" aria-controls="configuration-block-tabpanel-97a055193fa707f2025ee25bb469c74aa29b2c6c" aria-selected="true" data-active="true">
            <span>Webpack Encore</span>
        </button>
        <button role="tab" type="button" data-language="assetmapper" aria-controls="configuration-block-tabpanel-1865de6b2b585de79e8700d7968050722e562c44" aria-selected="false" tabindex="-1">
            <span>AssetMapper</span>
        </button>
    </div>

    <div role="tabpanel" id="configuration-block-tabpanel-97a055193fa707f2025ee25bb469c74aa29b2c6c" aria-label="Webpack Encore" class="configuration-codeblock" data-language="webpack-encore" style="">
        <p>Webpack Encore stuff!</p>
        <div translate="no" data-loc="1" class="notranslate codeblock codeblock-length-sm codeblock-yaml">
            <div class="codeblock-scroll">
                <pre class="codeblock-lines">1</pre>
                <pre class="codeblock-code"><code><span class="hljs-attr">I am yaml:</span></code></pre>
            </div>
        </div>
    </div>
    <div role="tabpanel" id="configuration-block-tabpanel-1865de6b2b585de79e8700d7968050722e562c44" aria-label="AssetMapper" class="configuration-codeblock" data-language="assetmapper" style="display: none">
        <p>AssetMapper stuff!</p>
        <p>And another paragraph.</p>
    </div>
</div>
